src/app/components/articles/article/article.component.scss

```diff
@@ -0,0 +1,31 @@
+@use "/src/styles.scss" as *;
+
+* {
```

Comment:
remove this part since it has been integrated in global scss file

Reply:
Done.